frontend/src/Room.vue

```diff
@@ -3,7 +3,7 @@
 
   <div v-if="room" ref="room" class="room" :class="{ mobile }">
     <div class="text">
-      <div data-autotest="displayedRoomName" v-if="room.name">Room: {{ room.name }}</div>
+      <div data-autotest="displayedRoomName" v-if="room.name">Room: {{ room.name }}, connectionType: {{ conferenceRoomType }}</div>
```

Comment:
Not to confuse developers, we should use the same terminology as in the public API for creating rooms, that is "conferenceType " as shown in https://sdk.virtualcareservices.net/sdks/rest/#create-a-room.